frontend/src/components/organisms/forms/participateForm.tsx

```diff
@@ -12,10 +13,11 @@ const Component = (props: { event?: any; callback: () => void }) => {
     setError(false);
     participationApi
       .createParticipation(props.event.id)
-      .then(() => {
+      .then((response) => {
         if (closeModalRef.current) {
           closeModalRef.current();
         }
+        window.open(API_URL+`/participation/${response.data.id}/ticket`,'_blank');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ISPP-Eventia/eventus/pullRequest?prid=9114788)
```suggestion
        window.open(API_URL+`/participation/${response.data.id}/ticket`,"_blank");
```